shared/templates/template_OVAL_yamlfile_value

```diff
@@ -49,9 +49,9 @@
   </ind:yamlfilecontent_object>
 
   <ind:yamlfilecontent_state id="state_{{{ rule_id }}}" version="1">
-    <ind:value datatype="record"{{% if ENTITY_CHECK %}} entity_check="{{{ ENTITY_CHECK }}}"{{% endif %}}>
+    <ind:value datatype="record">
```

Comment:
This patch is already in master and should be dropped from this PR (git rebase origin/master && git push should do)